cuebot/src/test/java/com/imageworks/spcue/test/dao/postgres/DispatcherDaoTests.java

```diff
@@ -413,6 +413,40 @@ public void testfindUnderProcedJob() {
         boolean under = dispatcherDao.findUnderProcedJob(job1, proc);
         assertTrue(under);
     }
+
+    @Test
+    @Transactional
+    @Rollback(true)
+    public void testfindHigherPriorityJob() {
+        DispatchHost host = getHost();
+        JobDetail job1 = getJob1();
+        JobDetail job2 = getJob2();
+
+        jobDao.updateMinCores(job1, 0);
+        jobDao.updateMinCores(job2, 0);
```

Comment:
Do we need a test with other values for minCores too?

Reply:
`minCores` shouldn't effect this query at all, but we should be checking that this query is correctly bound to `maxCores`. I've also added both positive and negative tests for `higherPriorityJobExists`.